### PR TITLE
Spelling mistakes corrected, pending citation added and chapter12.dat renamed to ISFtable.dat

### PR DESCRIPTION
**Pull request by manpreet9112**

### CODE REVIEW

File: input/LimitStateDesign.tex [hunk 1]
```diff
@@ -436,48 +430,46 @@ \section{Unified Approach}
...
 
-$n_b (balanced), n_max, n_min$ are calculated by the \eqn 1.26 and are
+$n_b (balanced), n_{max}, n_{min}$ are calculated by the \eqn 1.26 and are
 given in \tablem 1.2 for ready reference. Balanced failure is given by
 $\epsilon_1 = 0.0035, \epsilon_{st} =\epsilon_{sy}$ ($\epsilon_{sy}$
 corresponds to a steel strain when $f_{st} = 0.87 f_y$, to be read from
 stress-strain curves for steel given in \fig 23 of the
-\citetitle{is4562000}, balanced failure modiﬁed for ductility is given for
-$\epsilon_1 = 0.0035, \epsilon_{st} = \epsilon-{std}$ and tension failure
+\citetitle{is4562000}, balanced failure modified for ductility is given for
```
Comment (mandeeps708) [inline input/LimitStateDesign.tex:464]: Can't see the difference between the two. Both seem to be "modified".

File: input/LimitStateDesign.tex [hunk 2]
```diff
@@ -503,58 +496,66 @@ \section{Unified Approach}
...
-in accordance with \citetitle{is4562000}. For each type of structural member, only
+design of all critical sections in structural members.
+
+
+Design aids-tables and charts-are developed in subsequent  chapters for
+easy and quick design of structural members—slabs, beams, columns and footings—based
+on the limit state method in accordance with \citetitle{is4562000}. For each type of structural member, only
 one type of design aid is given, which is chosen on the basis of its
-effectiveness and convenience in practice. All design aids have been 
+effectiveness and convenience in prac tice. All design aids have been
```
Comment (mandeeps708) [inline input/LimitStateDesign.tex:549]: practice?

File: input/LimitStateDesign.tex [hunk 3]
```diff
@@ -585,31 +587,29 @@ \section{Analysis and Design of Structures}
 reinforcement in all members of buildings with the following two exceptions.
 
 \begin{enumerate}[(i)]
-
-
 \item Reinforcement in Slabs
 
-Mild steel plain bars (Fe 250) should be used in slabs in order to reduce
+Mild steel plain bars (\fetwofivezero8) should be used in slabs in order to reduce
```
Comment (mandeeps708) [inline input/LimitStateDesign.tex:592]: Something wrong with `\fetwofivezero`?

File: input/LimitStateDesign.tex (hunk 1 above)
Comment (manpreet9112) [inline input/LimitStateDesign.tex:464]: I tell you. The first word modified was copy paste from .txt file that we have created using OCR. So, it was printed like this "modied" in pdf. you can see there is no space between "fi". So, I write it manually, Now its Ok.  It show complete word modified in pdf.

File: input/LimitStateDesign.tex (hunk 2 above)
Comment (manpreet9112) [inline input/LimitStateDesign.tex:549]: Yeah! I have seen it after  commit. Can I correct it in next PR?.

File: input/LimitStateDesign.tex (hunk 3 above)
Comment (manpreet9112) [inline input/LimitStateDesign.tex:592]: Yes, 8 is written by mistake. This will be corrected in  next PR.

File: input/LimitStateDesign.tex (hunk 1 above)
Comment (mandeeps708) [inline input/LimitStateDesign.tex:464]: Yes, it seems to be congested a bit.

File: input/LimitStateDesign.tex (hunk 2 above)
Comment (mandeeps708) [inline input/LimitStateDesign.tex:549]: Yes, definitely.

File: input/LimitStateDesign.tex (hunk 3 above)
Comment (mandeeps708) [inline input/LimitStateDesign.tex:592]: OK. I am closing this one. You make changes and send a new one.

**Pull request closed without merging**